### PR TITLE
fix:faq section

### PR DESCRIPTION
**Pull request by route-2**
I have added a few extra questions to the docs, in the user section as well as developer section.

### CODE REVIEW

File: src/modules/faq/FaqModule.tsx [hunk 1]
```diff
@@ -373,6 +428,75 @@ function FaqModule() {
...
+                <Span color={theme.color}> How do I integrate Video? </Span>
+                <BsChevronExpand size={20} color={'#ddd'} />
+              </Question>
+
+              {showAnswers[18] && (
+                <Answer>
+                  <Span>
+                  You can integrate chat functionality by following the instructions outlined in this documentation: {" "}
+                    <AMod target="_blank" href="https://push.org/docs/chat">
+                      Push (EPNS) Documentation for Chat
```
Comment (HarshRajat) [inline src/modules/faq/FaqModule.tsx:493]: Should be Video specific

File: src/modules/faq/FaqModule.tsx [hunk 2]
```diff
@@ -373,6 +428,75 @@ function FaqModule() {
...
+                <Span color={theme.color}> How do I integrate Video? </Span>
+                <BsChevronExpand size={20} color={'#ddd'} />
+              </Question>
+
+              {showAnswers[17] && (
+                <Answer>
+                  <Span>
+                  You can integrate chat functionality by following the instructions outlined in this documentation: {" "}
+                    <AMod target="_blank" href="https://push.org/docs/video">
+                      Push (EPNS) Documentation for Chat
```
Comment (HarshRajat) [inline src/modules/faq/FaqModule.tsx:470]: Duplicate, I think you meant spaces over here

File: src/modules/faq/FaqModule.tsx [hunk 3]
```diff
@@ -281,31 +330,37 @@ function FaqModule() {
...
                 }}
                 hover="#e20880">
                 <Span color={theme.color}>How do I send notifications?</Span>
                 <BsChevronExpand size={20} color={'#ddd'} />
               </Question>
 
-              {showAnswers[9] && (
+              {showAnswers[12] && (
                 <Answer>
-                  <Span>Sending a notification is possible from the DAPP, Back-end as well as Smart Contracts.</Span>
```
Comment (HarshRajat) [inline src/modules/faq/FaqModule.tsx:293]: Rework this to point to the correct documentation at push.org/docs

**Review by HarshRajat — CHANGES_REQUESTED** [review]
Screenshot 2023-12-20 at 7 26 42 PM

These are outdated as we have unified SDK now, please redo

**Review by 0xNilesh — APPROVED** [review]
just resolved few typos, rest LGTM

**Pull request merged**